lib/l10n/intl_en.arb

```diff
@@ -173,6 +178,7 @@
   "messageResetPassword": "Enter your e-mai in order to receive instructions on how to reset your password.",
   "messageDeleteAccount": "Are you sure you want to delete your account?",
   "messageCannotBeUndone": "This action cannot be undone.",
+  "messageChangeEmail": "Are you sure you want to change the email to {email}@stud.acs.upb.ro?",
```

Comment:
Include the mail domain from the code, maybe we need to change it in the future. There's already a string constant for it.
```suggestion
  "messageChangeEmail": "Are you sure you want to change the account email to {email}?",
```